source/1nce_zero_touch_provisioning.c

```diff
@@ -596,5 +635,106 @@ char * str_replace( char * orig,
     strcpy( tmp, orig );
     return result;
 }
+/*-----------------------------------------------------------*/
+#ifdef democonfigRANGE_SIZE
+    static int response_length( char * PART )
+    {
+        int pch = strstr( PART, "bytes" );
+
+        int rangeStart, rangeEnd, rangeOriginalSize;
+
+        if( 3 == sscanf( pch,
```

Comment:
This if condition has no line of code in the bracket. What is the purpose of this if condition?